fixture-monkey-api/src/main/java/com/navercorp/fixturemonkey/api/introspector/BeanArbitraryIntrospector.java

```diff
@@ -117,6 +117,10 @@ private Function<Map<ArbitraryProperty, Object>, Object> combine(
 							writeMethod.getName(),
 							value,
 							ex);
+					} catch (NullPointerException ex) {
+						LOGGER.warn("The given property '{}' may not have a setter",
```

Comment:
provides more information, `Types.getActualType(property.getType())`, would be helpful.

The user can see which classes have no setter.